pylsl/pylsl.py

```diff
@@ -1291,7 +1291,20 @@ def resolve_stream(*args):
 except:
     print("pylsl: ContinuousResolver not (fully) available in your liblsl "
           "version.")
-        
+
+# int64 support on windows and 32bit OSes will be properly implemented starting
+# with library version 1.14
+if bitness != 32 and os_name != 'Windows' or lib.lsl_library_version() > 113:
```

Comment:
Please remove the library version check until integer handling is actually merged into liblsl and released.

Reply:
Done.